users/d4mation/config.h

```diff
@@ -0,0 +1,13 @@
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```

Comment:
If you use `LTO_ENABLE = yes`, you don't need these.  In fact, it may error out for redefining it.